transcom/mymove
# B-19934 SSW Display Signed Certifications

## [B-19934](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19934)

## Summary

[B-19933](https://github.com/transcom/mymove/pull/12844) was made and merged upstream of this, it adds additional signed certifications to the database when a service counselor approves an AOA and when they close out a PPM. This story retrieves and displays those signed certifications on the AOA/PPM packet cumulatively. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the customer application, create a PPM with an AOA Request
2. When you go to submit the PPM, sign it with a different name than the name associated with your test data. This will prove where a piece of data is being pulled from later
3. Access the office application, login as a Service Counselor, approve the AOA Request, orders, and shipment
4. Access the customer application again as the previous user, return to your previous move
5. Download the AOA Packet
6. Ensure that the Signatures section includes the typed name input from step 2 for the customer signature, as well as the Service Counselor's name under the AOA section of the PPPO/PPSO signatures. The SSW should not have a signature next to it yet. Additionally ensure that the signature date for the AOA is listed in the date section, with the time removed. There should be no date for the SSW section yet. See the screenshot below for reference, though note it has SSW data included.
7. As the customer, continue through the PPM closeout process and upload any attachments you wish to include. Sign and submit them to the Service Counselor again.
8. Log back in as the service counselor, approve the documents and closeout the PPM.
9. As the service counselor or customer, download the Payment Packet.
10. Verify the information from step 6, but now there should be the closeout counselor's signature next to the SSW section, as well as the date it was signed.

### Backend


## Screenshots
<img width="569" alt="image" src="https://github.com/transcom/mymove/assets/148906585/f91f1c94-013d-4e0c-a48b-f8a7f98706f3">
